styles/blue2_theme.css

```diff
@@ -134,3 +136,14 @@ a:hover {
   /* thanks https://codepen.io/sosuke/pen/Pjoqqp for generating this filter (the first two properties set the icon to black first to achieve a truer color match) */
   filter: brightness(0) saturate(100%) invert(25%) sepia(11%) saturate(6019%) hue-rotate(315deg) brightness(97%) contrast(89%);
 }
+
+
+
+/***** Page-specific: program.php *****/
+div.mutualEvent {
+  background-color: rgba(112, 112, 158, 0.1);
+}
+
+div.mutualEvent h4 {
```

Comment:
changed color per Yuval's comment about distinguishing session title from paper titles and authors more